test/powershell/Modules/Microsoft.PowerShell.Utility/Get-Alias.Tests.ps1

```diff
@@ -155,6 +155,29 @@ Describe "Get-Alias DRT Unit Tests" -Tags "CI" {
             $returnObject[$i].Definition | Should -Be 'Get-Command'
         }
     }
+
+    # Below test was suggested by GitHub Copilot to aid with fixing issue #25616
+    It "Get-Alias DisplayName should always show AliasName -> ReferencedCommand for all aliases" {
+    # Clean up any existing aliases
+    if (Get-Alias -Name Test-MyAlias -ErrorAction SilentlyContinue) { Remove-Item Alias:Test-MyAlias }
+    if (Get-Alias -Name tma -ErrorAction SilentlyContinue) { Remove-Item Alias:tma }
```

Comment:
No need to check aliases before removing - Remove-Itrm -ErrorAction SilentlyContinue should work well.